flask_restful/__init__.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, app=None, prefix='',
                  default_mediatype='application/json', decorators=None,
                  catch_all_404s=False, serve_challenge_on_401=False,
                  url_part_order='bae', errors=None):
-        self.representations = dict(DEFAULT_REPRESENTATIONS)
+        self.representations = OrderedDict(DEFAULT_REPRESENTATIONS)
```

Comment:
Yes, I left it as `{'application/json': output_json}` on the assumption that would never change since
> [Out of the box, Flask-RESTful is only configured to support JSON.](http://flask-restful-cn.readthedocs.org/en/0.3.4/extending.html#content-negotiation)

I'll update that now